portx/modules/ROOT/pages/availability.adoc

```diff
@@ -2,10 +2,14 @@
 
 You can configure windows (periods of time) during which partners or transactions are on-hold or inactive.
 
-include::{partialsdir}/edit-settings.adoc[]
+include::{partialsdir}/relationships-settings.adoc[]
 [start=3]
 
-. In the *CONFIGURATION* section of the left-hand navigation pane on the <<partner-configuration.adoc#img-company-information, Company Information Page>>, click *Availability*.
+. Select a relationship with partners and/or applications, from the list. +
```

Comment:
What constitutes a relationship is kind of complicated, but one thing it cannot contain is a partner _and_ an application. So, here, I'd say:

Select a **_Relationship_** with which you want to work from the list. For more information about Relationships. see [Actors, Relationships and Artifacts](https://docs.portx.io/integration-hub/latest/actors-relationships-and-artifacts.html).

Notes:
* Bold italics for the first occurrence of Relationship, and link it to the letter R in the Glossary (see other glossary terms for examples of how to do this).
* Capitalize Relationship because, as we use it in a way that's specific to us, it's a proper noun.
* Use the xref method to link to the _Actors_ et al. Page, not the http: method I put there. 
* For your own education, check out that page and also the different scenarios under Routing Scenarios. There's a lot of useful info about how Relationships work in there.

Reply:
This change is now applied to all applicable configuration docs.

Reply:
Thx - in the end I concluded that this text would best appear at the top of the Relationships partial, I hope you concluded the same :-)